src/components/Image/docs/index.story.tsx

```diff
@@ -11,15 +14,17 @@ import {
   title,
 } from 'wix-storybook-utils/Sections';
 import {
-  Image,
-  ResizeOptions,
   AspectRatioPresets,
+  Image,
   LoadingBehaviorOptions,
+  ResizeOptions,
 } from '../';
+import { StoryCategory } from '../../../../stories/storyHierarchy';
 import { allComponents } from '../../../../stories/utils/allComponents';
 import { settingsApi } from '../../../../stories/utils/SettingsApi';
+import { settingsPanel } from '../../../../stories/utils/SettingsPanel';
 import * as examples from './examples';
-import { StoryCategory } from '../../../../stories/storyHierarchy';
+import { ImageWiringExample } from './ImageWiringExample';
 
```

Comment:
Jonathan created an OptimizedStylesBanner for all the components that use the overrideStylesParams.
Since this component was created before he updated it in the generator we should add it manually. 

`import { OptimizedStylesBanner } from '../../../../stories/OptimizedStylesBanner';`

` header({
    component: <OptimizedStylesBanner />,
}),`